controller/backup_controller.go

```diff
@@ -282,18 +282,18 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 			backupVolume != nil && backupVolume.DeletionTimestamp == nil {
 			backupTargetClient, err := newBackupTargetClientFromDefaultEngineImage(bc.ds, backupTarget)
 			if err != nil {
-				log.WithError(err).Error("Error init backup target clients")
+				log.WithError(err).Warn("Failed to init backup target clients")
 				return nil // Ignore error to prevent enqueue
 			}
 
 			if unused, err := bc.isBackupNotBeingUsedForVolumeRestore(backup.Name, backupVolumeName); !unused {
-				log.WithError(err).Warnf("Unable to delete remote backup")
+				log.WithError(err).Warn("Unable to delete remote backup")
```

Comment:
`Failed to check volume restore usage of the backup`

Reply:
The function return `unused == false`
Not only when the backup is used but if there is any error during checking
So the message here needs to keep general